yt/frontends/ramses/data_structures.py

```diff
@@ -233,7 +245,13 @@ def _read_particle_header(self):
         self.particle_header = hvals
         self.local_particle_count = hvals['npart']
 
-        particle_fields = [
+        # Try reading particle file descriptor
+        if self._has_part_descriptor:
+            particle_fields = (
+                _read_part_file_descriptor(self._part_file_descriptor))
+            ptype = 'io'
```

Comment:
Maybe at this point issue a logging message suggesting to do `ds.add_ptypes()` to automatically load the particle types in.

Reply:
Not necessary with latest commit.